src/main/java/com/artbeatte/exercises/memorygame/MemoryGameBoard.java

```diff
@@ -38,13 +39,21 @@ public MemoryGameBoard(int width, int height) {
         mFlippedIndex = -1;
     }
 
-    public Card flipCard(int x, int y) {
-        if (x < 0 || x >= mWidth || y < 0 || y >= mHeight) {
+    public int getWidth() {
+        return mWidth;
+    }
+
+    public int getHeight() {
+        return mHeight;
+    }
+
+    public Card flipCard(int width, int height) {
+        if (width < 0 || width >= mWidth || height < 0 || height >= mHeight) {
             throw new IndexOutOfBoundsException(String.format("tried to access (%d,%d) on board of size (%d,%d)",
-                    x, y, mWidth, mHeight));
+                    width, height, mWidth, mHeight));
```

Comment:
Confusing name change - coordinate variable name convention is X/Y, not Width/Height. Those are bounds.